lib/assessment.js

```diff
@@ -8,15 +8,39 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
+var min = function min(array, callback, initialValue) {
```

Comment:
The question is asking us to create a callback function `min` to pass into reduce as an argument; i.e.
`[1,2,3].reduce(min)`

Here's an example of one that might work.
```javascript
var min = function(previous, current) {
  if (previous < current) {
    return previous;
  }  else {
    return current;
  }
};
```